client/src/App.js

```diff
@@ -16,35 +16,38 @@ class App extends Component {
   componentDidMount() {
     fetch('/api/v1/stream')
         .then((response) => {
-            if(response.status === 200) {
-                return response.json()
-            }})
+          if (response.status === 200) {
+            return response.json()
+          }
+        })
         .then((streamData) => {this.setState({streamData})})
   }
 
   render() {
     if (this.state.streamData == null) {
-        return null;
+      return null;
     }
     return (
-      <div className='App'>
-        <Header />
-        {this.state.streamData.clusters.map((cluster) => {
-            switch(cluster.type) {
-            case "AppOfTheWeekCluster":
-                return <AppOfTheWeek data={cluster} key={cluster.id} />
-            case "StandardCluster":
-                return <StandardCluster data={cluster} key={cluster.id} />
-            case "TopCharts":
-                return <TopCharts data={cluster} key={cluster.id} />
-            case "RecommendedCategories":
-                return <RecommendedCategories data={cluster} key={cluster.id} />
-            default:
-                return null;
-        }
-        })}
-        <Footer />
-      </div>
+        <div className='app'>
+            <Header />
+            <div className='clusters'>
+                {this.state.streamData.clusters.map((cluster) => {
+                    switch (cluster.type) {
+                        case 'AppOfTheWeekCluster':
+                            return <AppOfTheWeek data = {cluster} key = {cluster.id} />
```

Comment:
Can you remove the spaces inbetween the prop key and the value

`data={cluster} key={cluster.id}`